src/util/serialise/Serialise.hpp

```diff
@@ -46,22 +46,21 @@ namespace util {
         template <typename T, typename Check = T>
         struct Serialise;
 
-        // Plain old data
+        // Trivially copyable data
         template <typename T>
-        struct Serialise<T, std::enable_if_t<std::is_trivial<T>::value, T>> {
+        struct Serialise<T, std::enable_if_t<std::is_trivially_copyable<T>::value, T>> {
 
             static inline std::vector<char> serialise(const T& in) {
```

Comment:
Or would you rather keep the `char` for windoze crap?
```suggestion
            static inline std::vector<uint8_t> serialise(const T& in) {
```